spanner/api/Spanner.Samples/UpdateDataWithJsonAsync.cs

```diff
@@ -0,0 +1,97 @@
+﻿// Copyright 2021 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START spanner_update_data_with_json_column]
+
+using Google.Cloud.Spanner.Data;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+public class UpdateDataWithJsonAsyncSample
+{
+    public class Venue
+    {
+        public int VenueId { get; set; }
+        public string VenueDetails { get; set; }
+    }
+
+    public async Task UpdateDataWithJsonAsync(string projectId, string instanceId, string databaseId)
+    {
+        List<Venue> venues = new List<Venue>
+        {
+            // If you are using .NET Core 3.1 or later, you can use System.Text.Json for serialization instead.
+            new Venue
+            {
+                VenueId = 19,
+                VenueDetails = JsonConvert.SerializeObject(new
+                {
+                    rating = 9,
+                    open = true,
+                })
+            },
+            new Venue
+            {
+                VenueId = 4,
+                VenueDetails = JsonConvert.SerializeObject(new object[]
+                {
+                    new
+                    {
+                        name = (string) null,
```

Comment:
Can't this be changed by an actual value. It's just a very serialization specific detail and it looks ugly.

Reply:
This `null` value is set to be consistent with the samples for the other clients.

@zoercai Any thoughts on this? Do we want to keep this `null` in all clients, or should we consider setting an actual value?

Reply:
Yep I'm fine with having this be an actual value. I'll change the sample code in the spec so that the name is "room 1".